specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/firewallPolicy.json

```diff
@@ -567,6 +567,256 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/firewallPolicies/{firewallPolicyName}/listIdpsSignatures": {
+      "post": {
+        "operationId": "FirewallPolicyIdpsSignatures_List",
+        "description": "Retrieves the current status of IDPS signatures for the relevant policy",
+        "parameters": [
+          {
+            "name": "parameters",
```

Comment:
could you move body at the end of the parameter list? Some old SDK codegen may have problems for 'body' at the beginning of the parameter list.

Reply:
Done